app/controllers/consent_forms/edit_controller.rb

```diff
@@ -0,0 +1,71 @@
+class ConsentForms::EditController < ConsentForms::BaseController
+  include Wicked::Wizard
+  include Wicked::Wizard::Translated # Defined custom URLs, see en.yml wicked
+
+  layout "two_thirds"
+
+  steps :name, :date_of_birth, :school
+
+  before_action :set_session
+  before_action :set_consent_form
+
+  def show
+    render_wizard
+  end
+
+  def update
+    case current_step
+    when :name
+      @consent_form.assign_attributes(name_params)
+    when :date_of_birth
+      @consent_form.assign_attributes(date_of_birth_params)
+    when :school
+      @consent_form.assign_attributes(school_params)
```

Comment:
What about rolling all the `*_params` methods back into `update_params` since we're using this approach? This method would be tidier since we only have to branch to take care of outliers, `school` and `date_of_birth` (for now).